pkg/util/log/clog.go

```diff
@@ -351,7 +359,7 @@ func (d *EntryDecoder) Decode(entry *Entry) error {
 			return io.EOF
 		}
 		b := d.scanner.Bytes()
-		m := entryRE.FindSubmatch(b)
+		m := entryCaptureRE.FindSubmatch(b)
```

Comment:
Rather than have a separate `entryCaptureRE`, I think we could find the index of the file:line match and strip of the prefix. Or perhaps change `entryMatchRE` to be `(?m)^([IWEF])(\d{6} \d{2}:\d{2}:\d{2}.\d{6}) (?:(\d+) )?([^:]+):(\d+)  `.